tst/EnergyPlus/unit/ChillerElectric.unit.cc

```diff
@@ -317,8 +317,6 @@ TEST_F(EnergyPlusFixture, ChillerElectric_WaterCooled_Simulate)
     Real64 curLoad = -10000.0;
     bool runFlag = true;
 
-    EnergyPlusData state;
```

Comment:
```state``` is a member variable of the test fixture. This shouldn't be redeclared as a local variable here.